services/ui-src/src/components/fields/TextAreaField.test.tsx

```diff
@@ -72,6 +76,40 @@ describe("<TextAreaField />", () => {
     });
   });
 
+  describe("Text area field hide condition logic", () => {
+    test("Text area field is hidden if its hide conditions' controlling element has a matching answer", async () => {
+      mockGetValues({
+        elements: [
+          {
+            answer: "yes",
+            type: "reportingRadio",
+            label: "Should we hide the other radios on this page?",
+            id: "reporting-radio",
+          },
+        ],
+      });
+      render(textAreaFieldComponent);
+      const textField = screen.queryByLabelText("test label");
+      expect(textField).toBe(null);
+    });
+
+    test("Text area field is NOT hidden if its hide conditions' controlling element has a different answer", async () => {
+      mockGetValues({
+        elements: [
+          {
+            answer: "idk",
+            type: "reportingRadio",
+            label: "Should we hide the other radios on this page?",
+            id: "reporting-radio",
+          },
+        ],
+      });
+      render(textAreaFieldComponent);
+      const textField = screen.queryByLabelText("test label");
```

Comment:
As you've seen, `queryByLabelText` will happily return `null` if the element cannot be found. In that case, the only error message Testing Library will be able to give us will be from the next line. Something like "Expected `null` to be visible, but it's not, because it's not even an element"

To get a more helpful error message out of TL, explaining what it failed to find and printing the markup where it failed to find it, we can make the tests slightly asymmetric: continue to use `queryByLabelText` in the hiding test, but use `getByLabelText` in this one.

```suggestion
      const textField = screen.getByLabelText("test label");
```